astropy/visualization/wcsaxes/core.py

```diff
@@ -195,17 +197,15 @@ def imshow(self, X, *args, **kwargs):
         elif origin == 'upper':
             raise ValueError("Cannot use images with origin='upper' in WCSAxes.")
 
-        # To check whether the image is a PIL image we can check if the data
-        # has a 'getpixel' attribute - this is what Matplotlib's AxesImage does
-
-        try:
+        if HAS_PIL:
             from PIL.Image import Image
-            from PIL.Image.Transform import FLIP_TOP_BOTTOM
-        except ImportError:
-            # We don't need to worry since PIL is not installed, so user cannot
```

Comment:
This comment looks helpful but is removed. Should we add it back?

Reply:
I don't think this comment is needed any more after the refactor.  Everything is now under a `if HAS_PIL` block so it should be obvious that it is used only if PIL is installed.  The docstring also explains the need to flip PIL images.